poetry/automerge.json5

```diff
@@ -0,0 +1,12 @@
+{
+  $schema: "https://docs.renovatebot.com/renovate-schema.json",
+  description: "poetry: automerge non major versions",
+  packageRules: [
+    {
+      matchManagers: ["poetry"],
+      automerge: true,
+      matchUpdateTypes: ["minor", "patch", "digest"],
+      automergeType: "pr",
```

Comment:
I assume you chose `automergeType: pr` to be able to allow for a quick revert if the change is breaking?

Reply:
yes